packages/apollo-provider/src/ApolloProvider.js

```diff
@@ -25,34 +23,34 @@ type ApolloProviderProps = {
  */
 class ApolloProvider extends PureComponent<ApolloProviderProps, ApolloProviderState> {
   state = {
-    client: null,
+    apolloClient: null,
   };
 
   async componentDidMount() {
     const { uri, getClient } = this.props;
     const schemaJson = await getFragmentsSchema(uri);
-    const client = getClient(schemaJson);
+    const apolloClient = getClient(schemaJson);
 
-    this.setState({ client });
+    this.setState({ apolloClient });
   }
 
   render() {
-    const { client } = this.state;
+    const { apolloClient } = this.state;
     const { children } = this.props;
-    const isClientCreated = !R.isNil(client);
+    const isClientCreated = !R.isNil(apolloClient);
     const isLoading = !isClientCreated;
 
     const rendered = typeof children === 'function'
```

Comment:
Why we should calculate render prop when `isClientCreated === false`?

Reply:
I removed the case with null from here to handle it in the consumer code